lib/kernel/doc/guides/eep48_chapter.md

```diff
@@ -145,8 +148,11 @@ information to each entry. This EEP documents the following metadata keys:
 - **`since := binary()`** - a binary representing the version such entry was
   added, such as `<<"1.3.0">>` or `<<"20.0">>`.
 
-- **`edit_url := binary()`** - a binary representing a URL to change the
-  documentation itself.
+- **`source := binary()`** - the location of the source file for this module.
+  Applies only to the module metadata.
+
+- **`source_anno := erl_anno:anno()`** - annotation metadata of the source
+  code (opposite to the annotation of the documentation attribute).
```

Comment:
I'm not sure what "annotation metadata of the source code" means. Can you give some example values?

Reply:
it is the Anno field (usually the second) in the Erlang Abstract Format. Do you have suggestions on how to describe it?